### ISSUE TITLE
New `1.1.0` release?

### ISSUE THREAD

**Issue by jtpio** [jupyter/jupyter#678]
JupyterLab was added to the metapackage in https://github.com/jupyter/jupyter/pull/615

But currently the latest release on PyPI is still `1.0.0`, without the `jupyterlab` dependency: https://pypi.org/project/jupyter

![image](https://user-images.githubusercontent.com/591645/229555022-6c894686-c84e-4c5b-8de4-2a28e5307acc.png)

With https://github.com/jupyter/jupyter/pull/628 it looks like it should now be possible to make releases from CI.

So maybe we should consider making a `1.1.0` release soon? Or even `2.0.0`.

This would be relevant for the upcoming JupyterLab 4 / Notebook 7 releases, expected before JupyterCon early May 2023.

**Comment by jtpio** [comment]
For reference the `jupyter` metapackage seems to be quite popular:

![image](https://user-images.githubusercontent.com/591645/229555938-7f85b398-8d87-419f-af87-c15c11c67180.png)

https://pepy.tech/project/jupyter

**Comment by jtpio** [comment]
cc @minrk @choldgraf since you both worked on https://github.com/jupyter/jupyter/pull/628.

What would it take to make a new release of the `jupyter` metapackage, so it includes `jupyterlab` added in https://github.com/jupyter/jupyter/pull/615?

Thanks for your help!

**Comment by choldgraf** [comment]
I'm not really sure unfortunately, I've never made a release of this and mostly am just doing whatever I can to help things move forward. If there is some action that I can help unblock please let me know.

**Comment by jtpio** [comment]
yeah I am not sure either. It's also not clear which Jupyter subproject owns this repository and the `jupyter` name on PyPI.

**Comment by choldgraf** [comment]
you seem to be one of the maintainers on pypi?

![image](https://user-images.githubusercontent.com/1839645/230388326-a312ba06-3f2c-4ce8-9324-1e503bce3f5f.png)

that's from: https://pypi.org/project/jupyter/

**Comment by jtpio** [comment]
Yes but I don't have commit rights on this repo (https://github.com/jupyter/jupyter). Which are usually necessary to push a new tag.

**Comment by choldgraf** [comment]
Ahhh I see - I think in that case you definitely should have commit rights if you are a maintainer on PyPI. I don't have them though ever since I was removed as an "owner status" from the `jupyter/` org. Maybe @jasongrout has them?

**Comment by jtpio** [comment]
So what should the process be? Continue the conversation here with @jupyter/software-steering-council, or open a new issue somewhere else?

**Comment by JohanMabille** [comment]
I think it makes sense to have this repo owned by the SSC and to give commit rights to the PyPI maintainers.

**Comment by choldgraf** [comment]
My 2 cents: I'd recommend that the SSC or the Foundations group is responsible for the _mandate_ and principles that guide how this repo evolves. But in terms of merge rights and day to day, I suggest we liberally give those rights to those with standing in any of the jupyter subprojects, under the assumption that they'll act in good faith within the principles defined by the SSC. To start, I think anybody with pypi access should have commit access.

**Comment by minrk** [comment]
@jtpio i added you to the repo

**Comment by minrk** [comment]
Finally published 1.1.0 with the updated dependencies (adds lab, removes qtconsole)
